package.json

```
@@ -13,6 +13,8 @@
     "clean": "tsc --build --clean",
     "dev": "tsc --build --watch",
     "lint": "eslint .",
+    "prettier:check": "prettier --check .",
+    "prettier:write": "prettier --write .",
```

[From Prettier 3.6](https://prettier.io/blog/2025/06/23/3.6.0#cli), we can use the [performance-improved CLI](https://prettier.io/blog/2023/11/30/cli-deep-dive).

```suggestion
    "prettier:check": "prettier --experimental-cli --check .",
    "prettier:write": "prettier --experimental-cli --write .",
```